torch/nn/__init__.pyi

```diff
@@ -1,6 +1,7 @@
+import builtins
```

Comment:
builtins is imported but doesn't seem to be used. Is this intentional?

Reply:
No I believe it came from a previous commit that I forgot to clean entirely, will remove